graphql/client.go

```diff
@@ -41,9 +41,15 @@ type Client interface {
 }
 
 type client struct {
-	httpClient Doer
-	endpoint   string
-	method     string
+	httpClient             Doer
+	endpoint               string
+	method                 string
+	withOperationNameParam bool
+}
+
+// WithOperationNameParam allows operationName to be included as a query parameter.
+func WithOperationNameParam(c *client) {
```

Comment:
please add a named type for the options so they show up nicely in the documentation